presto-main/src/main/java/io/prestosql/security/AccessControlModule.java

```diff
@@ -33,6 +34,8 @@ public void configure(Binder binder)
     {
         configBinder(binder).bindConfig(AccessControlConfig.class);
         binder.bind(AccessControlManager.class).in(Scopes.SINGLETON);
+        binder.bind(PrestoGroupProviderManager.class).in(Scopes.SINGLETON);
+        newOptionalBinder(binder, GroupProviderManager.class).setDefault().to(PrestoGroupProviderManager.class).in(Scopes.SINGLETON);
```

Comment:
We shouldn't need to do this as an optional binding, since `TestingPrestoServer` does not use `AccessControlModule`.